README.md

```diff
@@ -180,3 +180,17 @@ To query all the `@todo` items in Yard, the following command can be used:
 ```shell
 bundle exec yard list --query '@todo'
 ```
+
+### Adding Sub-documentation
+
+It is possible to add comprehensive documentation about the areas in each application. For example, in this repository, we have added a documentation on how volume tracking works. The process to add these sub-documentation is as follows.
+
+**Prerequesites**
+
+Please install `mkdocs` using `pip install mkdocs`.
```

Comment:
Does python version matter?

Reply:
I don't really know. The [doc](https://www.mkdocs.org/user-guide/installation/) instructs to install the "a recent version of Python".

Reply:
Ah ok, maybe just a comment saying a recent version of python as well in here.